### PR TITLE
[PPD-115] Payments: status check

### PR DESCRIPTION
**Pull request by aomegax**
#### List of Changes
- check on debt position status for paVerify and paGetPayments
- check on payment option status for paSendRT

#### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by pasqualespica — APPROVED** [review]
well done 💪

File: payments/src/main/java/it/gov/pagopa/payments/endpoints/validation/PaymentValidator.java [hunk 1]
```diff
@@ -39,11 +39,7 @@ public void isAuthorize(String ptIdDominioReq, String ptIdIntermediarioReq, Stri
             gpdClient.getOrganization(ptIdDominioReq);
         }  catch (Exception e) {
             log.error("[isAuthorize ERROR] error during GPD call" , e);
-            if (e.getCause() instanceof FeignException.FeignClientException) {
-                throw new PartnerValidationException(PaaErrorEnum.PAA_ID_DOMINIO_ERRATO);
-            } else {
-                throw new PartnerValidationException(PaaErrorEnum.PAA_SYSTEM_ERROR);
-            }
+            throw new PartnerValidationException(PaaErrorEnum.PAA_ID_DOMINIO_ERRATO);
```
Comment (jacopocarlini) [inline payments/src/main/java/it/gov/pagopa/payments/endpoints/validation/PaymentValidator.java:42]: why?

Comment (aomegax) [reply payments/src/main/java/it/gov/pagopa/payments/endpoints/validation/PaymentValidator.java:42]: according to this [workflow](https://pagopa.atlassian.net/wiki/spaces/PPD/pages/479987315/Controlli+payments)

Comment (jacopocarlini) [reply payments/src/main/java/it/gov/pagopa/payments/endpoints/validation/PaymentValidator.java:42]: but if there is a network problem the PAA_ID_DOMINIO_ERRATO it is not appropriate

Comment (aomegax) [reply payments/src/main/java/it/gov/pagopa/payments/endpoints/validation/PaymentValidator.java]: I agree with you...we need to modify again the workfow :)

**Pull request merged**